sentry/Cargo.toml

```diff
@@ -39,11 +39,14 @@ debug-logs = ["log_", "sentry-core/debug-logs"]
 # transports
 transport = ["reqwest", "native-tls"]
 reqwest = ["reqwest_", "httpdate", "tokio"]
-curl = ["curl_", "httpdate", "serde_json", "tokio"]
+curl = ["curl_", "httpdate", "tokio"]
 surf-h1 = ["surf_/h1-client", "httpdate"]
 surf = ["surf_/curl-client", "httpdate", "tokio"]
 native-tls = ["reqwest_/default-tls"]
 rustls = ["reqwest_/rustls-tls"]
+ureq = ["ureq_", "httpdate", "tokio"]
+ureq-tls = ["ureq", "ureq_/tls"]
+ureq-native-tls = ["ureq", "ureq_/native-tls"]
```

Comment:
Unfortunately these can't be added to `native-tls`/`rustls` as the `reqwest_/<feat>` dependency automatically turns on `reqwest_` too. Should we rename the `reqwest`-related features?

Reply:
hmm, while technically we don't guarantee api stability i don't think renaming features is that big an improvement that i'd change this now, it'll just create churn for people.